src/locales/fr.json

```diff
@@ -169,6 +169,9 @@
 "funksender": "Emetteur radio",
 "funksender_service_link_href": "mailto:info@bakom.admin.ch",
 "funksender_service_link_label": "info@bakom.admin.ch",
+"geodesy": "Geodesia",
```

Comment:
I have sent an email to @mariokeusen in order to correct translations. 
Should be done when he come back of his holidays.